app/src/main/java/net/skyscanner/backpack/demo/compose/NudgerStory.kt

```diff
@@ -72,6 +72,11 @@ fun NudgerStory(modifier: Modifier = Modifier) {
             modifier = Modifier.fillMaxWidth(),
         )
 
+        NudgerRowExample(
+            modifier = Modifier.fillMaxWidth(),
+            subtitle = "subtitle is an example of long text that can be wrapped",
```

Comment:
we generally use string resources - there's probably already a generic one that can be reused)

Reply:
Right! Have updating with above comment too, could you help to take a look again? Thanks!